data/mods/Xedra_Evolved/items/melee.json

```
@@ -559,5 +559,50 @@
     "melee_damage": { "cut": 13 },
     "weapon_category": [ "WHIPS" ],
     "passive_effects": [ { "id": "BLOODTHORNE_DRUID_HEARTWHIP" } ]
+  },
+  {
+    "type": "ITEM",
+    "id": "xe_troll_club",
+    "name": { "str_sp": "gigantic club" },
+    "description": "An enormous club, almost two and a half meters long, once wielded by a troll.",
+    "weight": "22600 g",
+    "longest_side": "250 cm",
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "any", "balance": "clumsy" },
+    "color": "brown",
+    "looks_like": "2h_club_kanabo_studded",
+    "symbol": "/",
+    "material": [ { "type": "wood", "portion": 1 } ],
+    "weapon_category": [ "GREAT_HAMMERS" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "BRUTAL", "SWEEP" ],
+    "qualities": [ [ "HAMMER", 1 ] ],
+    "flags": [ "DURABLE_MELEE", "ALWAYS_TWOHAND", "REACH_ATTACK", "NONCONDUCTIVE" ],
+    "volume": "20000 ml",
+    "price": "500 USD",
+    "price_postapoc": "5 USD",
+    "faults": [ { "fault_group": "handle_long" } ],
+    "melee_damage": { "bash": 44 }
+  },
+  {
+    "type": "ITEM",
+    "id": "xe_troll_giant_mace",
+    "looks_like": "mace_pipe",
+    "name": { "str": "gigantic mace" },
+    "description": "A weapon consisting of a wood handle with a heavy metal head.  This one was once wielded by a troll and is as long as a spear, with an enormous knob of metal thicker than your torso on the end.  You could probably smash down a door in a single blow with this, if you could wield it without collapsing under its weight.",
```

```suggestion
    "description": "A weapon consisting of a wood handle with a heavy metal head.  This one was once wielded by a troll and is as long as a spear, with an enormous knob of metal thicker than a human torso on the end.  You could probably smash down a door in a single blow with this, if you could wield it without collapsing under its weight.",
```

Made this change to reflect that you could be huge sized and that might not be true?  Or are trolls larger than even huge mutants?

---

They currently are much larger, mostly through breadth (a post-thresh bear mutant is 200 kg so only a quarter the size of a troll. I imagine the troll isn't really that much taller, though. But I agree it should still be changed since mutants could be pretty big.